classifier_and_summarizer/summarizer/text_media_matching/text_media_matching_interface.py

```diff
@@ -0,0 +1,31 @@
+''' Text Media  Matching interface '''
+from classifier_and_summarizer.summarizer.text_media_matching.text_media_matcher import \
+    TextMediaMatcher
+from classifier_and_summarizer.summarizer.text_media_matching.text_media_matching_preprocessor import \
+    TextMediaMatchingPreprocessor
+
+
+class TextMediaMatchingInterface:
```

Comment:
Why is there implementation in interface?

Reply:
Thanks for catching this. Completely missed that. Named it since it was the communication interface between the Summarizer and TextMediaMatcher . Will rename it to avoid this confusion.